src/configure/configure.ts

```diff
@@ -171,6 +171,7 @@ class Orchestrator {
                 else {
                     this.context['rightClickScenario'] = true;
                 }
+                this.context['resourceId'] = this.inputs.targetResource.resource.id;
```

Comment:
It will fail for non-webApp case ex AKS, because resource will be undefined there and it will throw an exception. 
Should we consider rightClickScenario=true for WebApp ??

Reply:
``
  private async getAzureResource(targetType: TargetResourceType) {
        var azureResourceSelector = ResourceSelectorFactory.getAzureResourceSelector(targetType);
        this.inputs.targetResource.resource = await azureResourceSelector.getAzureResource(this.inputs);
    }
``

We are using azureResourceSelector to get the resourceId. In case of AKS, it will be null because the flow is different for now., but it doesn't seem this will cause a problem?